app/controllers/api/v0/admins_controller.rb

```diff
@@ -36,6 +57,27 @@ def fetch_groups
         json_response(groups)
       end
 
+      def fetch_groups_by_approver_id
+        approver = Approver.find(params.require(:id))
+        groups = approver.groups
+
+        json_response(groups)
+      end
+
+      def fetch_requests_by_approver_id
+        approver = Approver.find(params.require(:id))
+        requests = approver.requests
+
+        json_response(requests)
+      end
+
+      def fetch_stages_by_approver_id
```

Comment:
We may not need an endpoint to list all stages by an approver. Listing of requests is enough.

Reply:
@bzwei I think instead of stages we should be calling this levels, since the feature is called multi   level approval. The stage means different steps in a single process.